risc0/circuit/rv32im/src/prove/emu/exec/mod.rs

```diff
@@ -220,7 +252,7 @@ impl<'a, 'b, S: Syscall> Executor<'a, 'b, S> {
             }
 
             if let Some(max_cycles) = max_cycles {
-                if self.cycles.user >= max_cycles as usize {
+                if self.cycles.user >= max_cycles {
```

Comment:
i noticed that we use usize and u64 for cycle count. might be good to converge on one but definitely something that can be done later